cloud_info_provider/providers/static.py

```diff
@@ -131,7 +131,8 @@ def get_instances(self, **kwargs):
     def get_compute_shares(self, **kwargs):
         fields = ('instance_max_cpu', 'instance_max_ram',
                   'instance_max_accelerators',
-                  'auth', 'sla', 'network_info', 'membership')
+                  'auth', 'sla', 'network_info', 'default_network_type',
+                  'public_network_name', 'membership')
```

Comment:
I think this should be enough and the OpenStack code above is not needed

Reply:
Yes, you are right, the openstack code was useless (was meant to be cutomized, but didn't customize anything). I've removed it.